packages/cli-repl/.depcheckrc

```diff
@@ -1 +1,10 @@
-ignores: ['lodash', 'moment']
+ignores:
+ - '@typescript-eslint/parser'
+ - '@typescript-eslint/eslint-plugin'
+ - chai
+ - sinon-chai
+ - sinon
+ - ts-sinon
```

Comment:
Opened https://jira.mongodb.org/browse/MONGOSH-1501 to at least deduplicate these two